src/PrestaShopBundle/Service/Cache/Refresh.php

```diff
@@ -55,7 +55,7 @@ public function __construct($env = 'prod')
         $this->commands = [];
 
         require_once _PS_ROOT_DIR_.'/app/AppKernel.php';
-        $kernel = new \AppKernel($this->env, true);
+        $kernel = new \AppKernel($this->env, false);
```

Comment:
I haven't done this because it was only used on the installer and currently cost too many resources.